mypy/checker.py

```diff
@@ -7724,7 +7729,9 @@ def warn_deprecated_overload_item(
                     candidate := item.func.type, CallableType
                 ):
                     if selftype is not None:
-                        candidate = bind_self(candidate, selftype)
+                        candidate = bind_self(
```

Comment:
I'm a bit surprised about this `bind_self`, actually. I cannot find where `inferred_dunder_get_type` gets passed to `bind_self`... I'm only looking at the code though. I'm certain it's used but... where?

Reply:
Hum, in `analyze_descriptor_access`, we pass `inferred_dunder_get_type` as `target`:

https://github.com/python/mypy/blob/9ff9946ca5476da00760e05519244a5adb070971/mypy/checkmember.py#L726-L728

but not sure what you mean exactly

Reply:
Like I'd expect some sort of symmetry: we repeat the actions we did to get target on each overload member. But I can't find where we `bind_self`.

Reply:
Like I'd expect some sort of symmetry: we repeat the actions we did to get target on each overload member. But I can't find where we `bind_self`.

---

To be specific, in `analyze_descriptor_access` (or in a function it calls) I expect a `bind_self` call as `dunder_get` turns into `inferred_dunder_get_type`... but I can't find one. Is it just not necessary?